src/push-build-information.ts

```diff
@@ -1,37 +1,20 @@
 import {InputParameters} from './input-parameters'
 import {info, setFailed} from '@actions/core'
-import {exec, ExecOptions} from '@actions/exec'
 import {context} from '@actions/github'
-import {promises as fsPromises} from 'fs'
-
-function getArgs(parameters: InputParameters): string[] {
-  info('🔣 Parsing inputs...')
-
-  const args = ['build-information']
-
-  if (
-    parameters.overwriteMode.length > 0 &&
-    parameters.overwriteMode !== 'FailIfExists'
-  ) {
-    if (
-      parameters.overwriteMode !== 'OverwriteExisting' &&
-      parameters.overwriteMode !== 'IgnoreIfExists'
-    ) {
-      setFailed(
-        'The input value, overwrite_mode is invalid; accept values are "FailIfExists", "OverwriteExisting", and "IgnoreIfExists".'
-      )
-    }
-    args.push(`--overwrite-mode=${parameters.overwriteMode}`)
+import {Client, ClientConfiguration} from '@octopusdeploy/api-client'
+
+async function getOctopusClient(parameters: InputParameters): Promise<Client> {
+  const config: ClientConfiguration = {
+    apiKey: process.env['OCTOPUS_CLI_API_KEY'],
+    apiUri: process.env['OCTOPUS_CLI_SERVER'],
```

Comment:
I am torn as to whether we should continue to have `CLI` in the environment variables. For consistency atm it sort of makes sense, but long term I think we'd be better to drop it. Thoughts?

Reply:
As we're not using the CLI, I think it'd be better to drop the `CLI` part of the environment variable as we'd have to go back and clean it up later once we convert the other actions...so might as well do it now as this is the new way.